recipe/meta.yaml

```diff
@@ -21,15 +26,15 @@ requirements:
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - {{ compiler('c') }}                    # [unix]
     - {{ compiler('cxx') }}                  # [unix]
-    - libprotobuf {{ version }}              # [build_platform != target_platform]
+    - libprotobuf {{ lib_major + "." + version }}  # [build_platform != target_platform]
```

Comment:
@xhochy, is this a solution you can live with?  It seems to make conda-smithy happy in the sense that it notices that we're pinning a specific version and doesn't include the version from conda-forge-pinning.